test/system-test/create.stest.ts

```diff
@@ -19,52 +19,48 @@ import { homedir } from 'node:os';
 import path, { join } from 'path';
 import { cwd } from 'process';
 import { DEFAULT_BUFFER_ENCODING } from '../../src/modules/constants';
-import { SdkConfig } from '../../src/modules/sdk';
-import { sdkDownloader } from '../../src/modules/package-downloader';
 
 const VELOCITAS_PROCESS = join('..', '..', '..', process.env['VELOCITAS_PROCESS'] ? process.env['VELOCITAS_PROCESS'] : 'velocitas');
 const TEST_ROOT = cwd();
 const VELOCITAS_HOME = `${homedir()}/.velocitas`;
 
 describe('CLI command', () => {
-    describe('create', () => {
-        beforeEach(async () => {
-            // When using this as real "system test"
-            // the next two lines are not necessary
-            const sdkConfig = new SdkConfig('python');
-            await sdkDownloader(sdkConfig).downloadPackage({ checkVersionOnly: false });
-            process.chdir(`${TEST_ROOT}/testbench/test-create`);
-            copySync('./sdk', `${VELOCITAS_HOME}/sdk/python/latest`);
-            process.chdir(`${TEST_ROOT}/testbench/test-create/vehicle-app-template`);
-        });
-        afterEach(() => {
-            process.chdir(`${TEST_ROOT}/testbench/test-create/vehicle-app-template`);
-            readdirSync(`${TEST_ROOT}/testbench/test-create/vehicle-app-template`).forEach((file) => {
-                const fileDir = path.join(`${TEST_ROOT}/testbench/test-create/vehicle-app-template`, file);
-                if (file !== 'package-index.json') {
-                    removeSync(fileDir);
-                }
-            });
-        });
-        it('should be able to create a project', async () => {
-            spawnSync(VELOCITAS_PROCESS, ['create', '-n', 'MyApp', '-l', 'python'], {
-                encoding: DEFAULT_BUFFER_ENCODING,
-            });
-
-            const creationConfigFile = readFileSync(
-                `${VELOCITAS_HOME}/sdk/python/latest/.project-creation/config.json`,
-                DEFAULT_BUFFER_ENCODING,
-            );
-            const creationConfig = JSON.parse(creationConfigFile);
-            const fileCheck: boolean[] = [];
-            creationConfig.files.forEach((file: string) => {
-                file = file.replace('.project-creation/', '');
-                fileCheck.push(existsSync(`${TEST_ROOT}/testbench/test-create/vehicle-app-template/${file}`));
-            });
-
-            expect(fileCheck.every((v: boolean) => v === true)).to.be.true;
-            expect(existsSync(`${TEST_ROOT}/testbench/test-create/vehicle-app-template/.velocitas.json`)).to.be.true;
-            expect(existsSync(`${TEST_ROOT}/testbench/test-create/vehicle-app-template/app/AppManifest.json`)).to.be.true;
-        });
-    });
+    // describe('create', () => {
```

Comment:
Given we use https://github.com/doosuu/pkg-velocitas-main for now, could we re-enable the tests?